src/main/scala/com/danielasfregola/twitter4s/entities/DirectMessageEvent.scala

```diff
@@ -0,0 +1,23 @@
+package com.danielasfregola.twitter4s.entities
+import java.time.LocalDateTime
+
+import com.danielasfregola.twitter4s.entities.enums.TweetType
+
+final case class DirectMessageEvent(`type`: TweetType.Value = TweetType.messageCreate,
+                                    id: DirectMessageId,
+                                    created_timestamp: LocalDateTime,
```

Comment:
`created_timestamp` cannot be a local date as we need to consider timezones as well. I suggest to change it to `Instant` and to add a default of `Instant.now`.